libs/ui-watt/src/lib/components/shell/nav-list/watt-nav-list.component.scss

```diff
@@ -0,0 +1,46 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+@use '@energinet-datahub/watt/utils' as watt;
+
+watt-nav-list {
+  --watt-nav-list-color-secondary: #f8ad3c;
```

Comment:
issue: The color value here should ideally come from a CSS Custom Property. I decided to use it like this because we don't currently have the color in our Design System. So once we update the colors, this place needs to be refactored.